src/components/Nav.vue

```diff
@@ -33,22 +33,35 @@
     >
 
       <div class="navbar-end">
-        <div class="navbar-item">
+        <div class="navbar-item" v-if="!getLog">
           <div class="buttons">
-            <router-link v-if="!getLog" to="/" class="button is-light">
+            <router-link to="/" class="button is-light">
               Se connecter
             </router-link>
-            <router-link v-if="getLog" to="/profile" class="button is-light">
-              Profile
-            </router-link>
-            <router-link v-if="getLog && hasUnconfiguredBackend" to="/create-account" class="button is-light">
+          </div>
```

Comment:
You can solve #57 if you want (optional !, this can be in another PR). Yes I just added it !

Reply:
Resolved